test/runnable_cxx/README.md

```diff
@@ -0,0 +1,23 @@
+# Tests for C++ interoperability
```

Comment:
For this file, you might want to add that they are run as part of the Github CI matrix and on a variety of C++ compilers.

Reply:
Good idea, done.